src/main/java/com/plivo/api/models/message/MmsMedia.java

```diff
@@ -1,33 +1,51 @@
 package com.plivo.api.models.message;
 
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
 import com.plivo.api.models.base.BaseResource;
 
+@JsonIgnoreProperties(value = {"id"})
```

Comment:
@renoldthomas-plivo Is this field removal expected as i can see comment like "Not Removing "id" field so that existing customers are not impacted" in your test plan doc

Reply:
Yes @Sajal-Singhal  This will impact only for **List Media using message_uuid** API. As we saw earlier, for this API I dont think any customer would be even using it as all fields were returning null except for URL in master branch. Hence removed the id.